beacon-chain/rpc/eth/helpers/error_handling.go

```diff
@@ -66,3 +68,38 @@ func PrepareStateFetchError(err error) error {
 	}
 	return fmt.Errorf("could not fetch state: %v", err)
 }
+
+func HandleGetBlockError(blk interfaces.ReadOnlySignedBeaconBlock, err error) error {
+	if invalidBlockIdErr, ok := err.(*lookup.BlockIdParseError); ok {
+		return status.Errorf(codes.InvalidArgument, "Invalid block ID: %v", invalidBlockIdErr)
+	}
+	if err != nil {
+		return status.Errorf(codes.Internal, "Could not get block from block ID: %v", err)
+	}
+	if err := blocks.BeaconBlockIsNil(blk); err != nil {
+		return status.Errorf(codes.NotFound, "Could not find requested block: %v", err)
+	}
+	return nil
+}
+
+func HandleGetBlockErrorJson(blk interfaces.ReadOnlySignedBeaconBlock, err error) *http2.DefaultErrorJson {
```

Comment:
Instead of creating this function, please use `WriteBlockFetchError` which is defined in `beacon-chain/rpc/eth/shared/errors.go`

Reply:
Yes! that function was not there when I wrote the initial refactor. Will use that one instead.